ome_zarr/reader.py

```diff
@@ -282,9 +282,9 @@ def __init__(self, node: Node) -> None:
                 "version", "0.1"
             )  # should this be matched with Format.version?
             datasets = multiscales[0]["datasets"]
-            # axes field was introduced in 0.3, before all data was 5d
-            axes = tuple(multiscales[0].get("axes", ["t", "c", "z", "y", "x"]))
-            if len(set(axes) - axes_values) > 0:
+            axes = multiscales[0].get("axes")
+            if version == "0.3" and len(set(axes) - axes_values) > 0:
+                # TODO: validate axis for > V0.3 ?
```

Comment:
We can probably define some scenarios that should be validated e.g. there are not enough spatial axes.

Can the `_validate_axes_types` method be re-used here?

Reply:
I'll reuse the full `_validate_axes()` to do validation for any version

Reply:
Done in bd16dfd